### ISSUE TITLE
Add mouse support to keypad

### ISSUE THREAD

**Issue by turingincomplete** [turingincomplete/MatchboxChip8#5]
Currently, the on-screen keyboard only listens to key events. This does not work on mobile/tablet for obvious reasons. The demo should be modified to allow users to click on the keypad buttons and trigger keypresses in the emulator.

**Comment by anaplian** [comment]
You have my sword.
